website/src/data/top-menu.yaml

```diff
@@ -0,0 +1,29 @@
+- menu-title: Composable Commerce
+  items:
+    - label: Getting Started
+      href: /../getting-started/
+    - label: Merchant Center
+      href: /../merchant-center
+    - label: Tutorials
+      href: /../tutorials
+    - label: HTTP API
+      href: /../api
+    - label: GraphQL API
+      href: /../api/graphql
```

Comment:
```suggestion
      href: https://docs.commercetools.com/api/graphql
```

Reply:
Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1